source/openapi3/split.yaml

```diff
@@ -1706,7 +1668,7 @@ tags:
 
       ```
 
-      The rejected, returned, voided & prefailed statuses are always accompanied by a failure code, title and detail as listed below.
+      The rejected, returned & voided statuses are always accompanied by a failure code, title and detail as listed below.
```

Comment:
I'd keep the oxford comma 😉 

```suggestion
      The rejected, returned, & voided statuses are always accompanied by a failure code, title and detail as listed below.
```

![image](https://user-images.githubusercontent.com/46548/169968122-2decf93e-7a0a-404b-9ac9-b875be2b81cc.png)